python/paddle/v2/fluid/io.py

```diff
@@ -11,6 +11,16 @@
 
 
 def is_parameter(var):
```

Comment:
As `is_parameter` is not in ` __all__ `, could it show correctly on the website?

Reply:
Yes, see screenshot below